### ISSUE TITLE
Question about training a text classifier

### ISSUE THREAD

**Issue by WeitaiKang** [ZzZZCHS/WS-3DVG#5]
Dear author, I have a question about the motivation to train a text classifier.

As claimed in your paper and provided in the code, you were computing the similarity between text classification score (trained using 18 classes) and the object classification score (obtained from pretrained groupfree). In your ablation study, the L_cls makes a significant improvement. My question is, given that you have the ground truth label of these 18 classes, why don't you use it to coarsely select the object. What makes this additional text classification training and the following similarity computing significant?

**Comment by WeitaiKang** [comment]
I am wandering if it is fair to use the object label when inference? If not, then if is it fair to use it when training?

**Comment by ZzZZCHS** [comment]
We don't think it is fair to use ground truth object labels during inference. Therefore, we need to train a text classifier during training. The object labels are used in training to serve as the supervision for the text classifier.
